### PR TITLE
Functional tests - Add new test 'BO > Logs > Log by email'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Add new test 'BO > Logs > Log by email'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | NIghtly => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4691301957
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/BObasePage.ts [hunk 1]
```diff
@@ -966,6 +969,15 @@ export default class BOBasePage extends CommonPage {
     return this.getTextContent(page, this.alertDangerBlockParagraph);
   }
 
+  /**
+   * Get alert block content
+   * @param page {Page} Browser tab
+   * @return {Promise<string>}
+   */
+  getAlertBlockContent(page: Page): Promise<string> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/BObasePage.ts]:
```suggestion
  async getAlertBlockContent(page: Page): Promise<string> {
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/pages/BO/BObasePage.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/BObasePage.ts]: Done :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**